.kitchen.yml

```diff
@@ -82,3 +82,22 @@ suites:
     encrypted_data_bag_secret_key_path: "test/integration/forwarder/encrypted_data_bag_secret"
     run_list:
       - recipe[elkstack::forwarder]
+
+<% if ENV['RACKSPACE_MONITORING_USERNAME'] && ENV['RACKSPACE_MONITORING_API_KEY'] %>
```

Comment:
I think that we don't have any alarm by default, so it should no create ticket ?
I'm happy to not set those ENV on the CI environment, which means you can test monitoring before to submit a PR with your own account, but CI will not actually do anything, but it sounds less robust